test/integration/docker-compose-java.yml

```diff
@@ -0,0 +1,82 @@
+version: '3.8'
+
+services:
+  javatestserver:
+    build:
+      context: ../..
+      dockerfile: test/integration/components/javatestserver/Dockerfile${TEST_DOCKERFILE_SUFFIX}
+    image: hatest-javatestserver
+    ports:
+      - "8085:8085"
+    environment:
+      LOG_LEVEL: DEBUG
+
+  javaautoinstrumenter:
+    build:
+      context: ../..
+      dockerfile: ./test/integration/components/otelauto/Dockerfile
+    command:
+      - /otelauto
+      - --config=/configs/instrumenter-config-java.yml
+    volumes:
+      - ./configs/:/configs
+      - ../../testoutput:/coverage
+      - ../../testoutput/run:/var/run/otelauto
+    image: hatest-javaautoinstrumenter
+    privileged: true # in some environments (not GH Pull Requests) you can set it to false and then cap_add: [ SYS_ADMIN ]
+    network_mode: "host"
```

Comment:
I had to use "host" mode for both the "pid" and the "network_mode". Without the network mode being host I wasn't getting the events. Without "pid" I was finding the process id of inside the container, 1 in case of the Java docker image, so the filtering didn't work.

Changing to host mode makes it a bit difficult to test the prometheus scraping, so I'm relying on our other scraping tests.